pyproject.toml

```diff
@@ -36,6 +36,10 @@ numpy = [
     {version = "^1.22.3", python = "^3.8"}
 ]
 prql-python = "^0"
+ipython = "7.33.0"
```

Comment:
I can't remember why I limited this initially...I think since we support python 3.7, and IPython > 8 requires python 3.8 or greater. But there are definitely better ways to specify this. I'll see what we can do